script.js

```diff
@@ -64,6 +64,17 @@ restaurantsApp.getRestaurantIDs = (searchTerm, searchLocation) => {
     });
 }
 
+// Use the GeoLocation-DB API to get the user's city
+restaurantsApp.getCity = async function() {
+    // wait for the result to come back (promise)
+    let location = await $.ajax({
+        url: "https://geolocation-db.com/jsonp",
+        jsonpCallback: "callback",
+        dataType: "jsonp"
+    });
+
+    return location.city;
+}
```

Comment:
can you explain why the getcity function is an async function?

Reply:
I wanted to `await` the AJAX call so I could return only the city from `getCity()`. In order to await an AJAX call, we have to make the outer function `async`.